aws/sdk/examples/qldb/src/bin/qldb-helloworld.rs

```diff
@@ -50,30 +49,26 @@ async fn main() -> Result<(), Error> {
         .unwrap_or_else(|| Region::new("us-west-2"));
 
     if verbose {
-        println!("OLDB client version: {}\n", qldb::PKG_VERSION);
-        println!("Region:              {:?}", &region);
-        println!("Ledger:              {}", ledger);
-
-        SubscriberBuilder::default()
-            .with_env_filter("info")
-            .with_span_events(FmtSpan::CLOSE)
-            .init();
+        println!("OLDB version: {}", PKG_VERSION);
+        println!("Region:       {:?}", &region);
+        println!("Ledger:       {}", ledger);
+        println!();
     }
 
     let conf = Config::builder().region(region).build();
     let client = Client::from_conf(conf);
+
     let result = client
         .send_command()
         .start_session(StartSessionRequest::builder().ledger_name(ledger).build())
         .send()
-        .await?;
+        .await
+        .expect("Could not start a session");
```

Comment:
we could probably be leaning towards `?` over `.expect()`. In general, we should be leading customers away from code that will panic